openlibrary/mocks/mock_ol.py

```diff
@@ -1,11 +1,16 @@
 import os
-import pytest
 import re
+
+import pytest
 import web
+try:
+    from web.browser import AppBrowser
+except ImportError:
```

Comment:
I don't think that the old form of the import is needed. I guess it doesn't hurt anything, but is there a reason to include it given that we don't support both web.py 0.39 and 0.40?